config/300-gitlabsource.yaml

```diff
@@ -107,19 +133,21 @@ spec:
                   type: object
               type: object
             eventTypes:
-              description: EventType is the type of event to receive from Gitlab.
-                These correspond to supported events to the add project hook https://docs.gitlab.com/ee/api/projects.html#add-project-hook
+              description: List of webhooks to enable on the selected GitLab
+                project. Those correspond to the attributes enumerated at
+                https://docs.gitlab.com/ee/api/projects.html#add-project-hook
               items:
                 enum:
-                - push_events
-                - push_events_branch_filter
-                - issues_events
                 - confidential_issues_events
+                - confidential_note_events
+                - deployment_events
+                - issues_events
+                - job_events
                 - merge_requests_events
-                - tag_push_events
                 - note_events
-                - job_events
                 - pipeline_events
+                - push_events
+                - tag_push_events
```

Comment:
did you actually sort them alphabetically :)

Reply:
Yes because that's how they appear in the docs, easier to know at a glance if we omitted one if the order is the same.

Reply:
Yes because that's how they appear in the docs, easier to know at a glance if we omitted one if the order is the same.